src/api_manager/fetch_metadata.h

```diff
@@ -19,17 +19,27 @@
 #include "src/api_manager/context/global_context.h"
 #include "src/api_manager/context/request_context.h"
 
+using ::google::api_manager::utils::Status;
```

Comment:
per style guide,  not to use "using" in the header files

Reply:
Done.